libs/positions/src/lib/positions-table.tsx

```diff
@@ -43,12 +51,15 @@ export interface MarketNameCellProps {
 }
 
 export const MarketNameCell = ({ valueFormatted }: MarketNameCellProps) => {
-  return valueFormatted ? (
-    <div className="leading-tight">
-      <div>{valueFormatted[0]}</div>
-      {valueFormatted[1] ? <div>{valueFormatted[1]}</div> : null}
-    </div>
-  ) : null;
+  if (valueFormatted && valueFormatted[1]) {
+    return (
+      <div className="leading-tight">
```

Comment:
chore (non blocking): I could see us needing a cell wrapper component to manage styles for cells with two rows. Perhaps create this and add it to react-helpers?

Reply:
I'm not sure about it. Creating two rows is rather trivial just add two block elements. It all depends what you want to put in those rows and how this will affect rowHeight